internal/backends/mcu/lib.rs

```diff
@@ -324,12 +324,18 @@ mod the_backend {
             }
         }
 
-        fn quit_event_loop(&self) {
-            EVENT_QUEUE.with(|q| q.borrow_mut().push_back(McuEvent::Quit));
-        }
+        fn event_loop_proxy(&self) -> Option<Box<dyn i_slint_core::platform::EventLoopProxy>> {
```

Comment:
I think it would be good of this function is called `new_event_loop_proxy()` or `create_event_loop_proxy()` to emphasise the allocative nature of it.